src/packages/eslint-plugin-eigenspace-styles/src/index.ts

```diff
@@ -0,0 +1,23 @@
+// @ts-ignore
+module.exports = {
```

Comment:
Given that there is no benefit from using typescript in this case, but a huge minus - in the form of a stage for assembly, I would not use typescript here.